scrapy/extensions/httpcache.py

```diff
@@ -226,7 +226,7 @@ def open_spider(self, spider):
         dbpath = os.path.join(self.cachedir, f'{spider.name}.db')
         self.db = self.dbmodule.open(dbpath, 'c')
 
-        logger.debug("Using DBM cache storage in %(cachepath)s" % {'cachepath': dbpath}, extra={'spider': spider})
+        logger.debug(f"Using DBM cache storage in {dbpath}", extra={'spider': spider})
```

Comment:
Old message from `pylint` related to this code line:
`scrapy/extensions/httpcache.py:229:21: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)`
New message from `pylint` related to this code line:
`scrapy/extensions/httpcache.py:229:8: W1203: Use lazy % or .format() or % formatting in logging functions (logging-fstring-interpolation)`